tailwind.config.js

```diff
@@ -0,0 +1,925 @@
+module.exports = {
+  content: ['./src/**/*.{html,vue}'],
+  // Used in the Labware component for slots, but colour is built dynamically so we need to make sure it's not purged
```

Comment:
I can't find the Labware component - is this your comment or copied over from STAN? Thanks

Reply:
That's something that needs to be removed. will do it.